IFC4x3/Properties/s/SequentialNumber_1rZx8VDW987QxAxZ64yNea/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SequentialNumber_1rZx8VDW987QxAxZ64yNea" Name="SequentialNumber" UniqueId="758fb21f-3602-481d-aeca-ee3184f17a24" PrimaryDataType="IfcLabel" />
```

Comment:
Number implies some sort of a number type not label.

Reply:
well im assuming it could be somethign like 5A possibly only change possibly could be SequenceCode maybe

Reply:
Change to code!

Reply:
ea970ecd2756e34e162c9f2a7fd9aceedd1ac5f3